assets/zstandard/main.py

```
@@ -0,0 +1,111 @@
+
+    def ingest(self):
+        """
+        Ingests the dataset at self.datasets_path
+        """
+        self.docker_execute(f"zstd -3 -o {ZST_FILE_PATH} {self.datasets_path}")
+    
+    def search(self, query):
+        """
+        Searches an already-ingested dataset with query, which is populated within config.yaml
+        """
+        assert query == "notasearch"
```

Ruff (0.12.2)

Use of `assert` detected

(S101)